lib/src/install.rs

```diff
@@ -865,37 +865,42 @@ pub(crate) fn setup_tmp_mounts() -> Result<()> {
 #[context("Ensuring sys mounts")]
 pub(crate) fn setup_sys_mounts() -> Result<()> {
```

Comment:
Should this just become a function that takes a /sys subpath instead of looping? That seems to make more sense now that I look at it... Or as you said, we can just drop the other code that does this, and just make sure this is called ahead of where that was done before.